### PR TITLE
Made it work with npmjs.com!!!1

### PR DESCRIPTION
**Pull request by marksyzm**
Hey, just made some changes to this and realised you're just down the road! I should have come by on my bike rather than writing this PR :grin: 

Can you wrap these changes in and publish this angular module to npmjs.org please? That way I can use it with webpack/browserify (as can everyone else) - thanks!

### CODE REVIEW

File: angular-bind-html-compile.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 (function (angular) {
     'use strict';
 
-    var module = angular.module('angular-bind-html-compile', []);
+    var mod = angular.module('angular-bind-html-compile', []);
```
Comment (wytrych) [inline angular-bind-html-compile.js:4]: Maybe `bindHtmlCompile` would be a better name?

**Comment by wytrych** [comment]
Could you merge master here, so we could merge it?

File: angular-bind-html-compile.js (hunk 1 above)
Comment (marksyzm) [inline angular-bind-html-compile.js:4]: That's a breaking change for anyone using it via npm though, so you'll have to jump up a version

Comment (wytrych) [inline angular-bind-html-compile.js:4]: I'd say it's worth it though. Could you update the changelog in this PR ?

**Comment by adam-beck** [comment]
What's the status on this? Any reason it isn't being merged that I can help with?

**Comment by marksyzm** [comment]
Well, I'm currently on holiday in France, but to be honest I forgot all about it so you're welcome to shoot

**Comment by adam-beck** [comment]
Enjoy your holiday!

**Comment by zwacky** [comment]
having this on npm would be sweet _bump_

in the meantime:

```
"dependencies": {
    "angular-bind-html-compile": "git+https://github.com/incuna/angular-bind-html-compile.git",
    ...
```

**Comment by marksyzm** [comment]
I'll merge master in in a mo

**Comment by evankleist** [comment]
Bump?

**Comment by wytrych** [comment]
Since I have push access now I will fix up this PR

**Pull request merged**